Sasquatch/Sasquatch/ViewControllers/MSDocumentDetailsViewController.swift

```diff
@@ -72,6 +74,31 @@ class MSDocumentDetailsViewController: UIViewController, UITableViewDelegate, UI
     self.presentingViewController?.dismiss(animated: true, completion: nil)
   }
 
+  @IBAction func refreshButtonClicked(_ sender: Any) {
+    var partition = documentContent?.partition ?? ""
+    if (partition.contains("user")) {
+        partition = "user"
```

Comment:
Should use `kMSDataUserDocumentsPartition` :)

Reply:
In the check too actually.

Reply:
You can create a constant in the AppCenterDelegate that would reflect the AppCenter Data one. That will avoid the Puppet/Sasquatch import nightmare.